pkg/util/steps/runner.go

```diff
@@ -54,6 +56,14 @@ func Run(ctx context.Context, log *logrus.Entry, pollInterval time.Duration, ste
 
 		if err != nil {
 			log.Errorf("step %s encountered error: %s", step, err.Error())
+			if strings.Contains(err.Error(), "ThrottlingLimitExceeded") {
+				err = api.NewCloudError(
+					http.StatusTooManyRequests,
+					api.CloudErrorCodeThrottlingLimitExceeded,
+					"",
+					"Requests are being throttled due to Azure Storage limits being exceeded. Please visit https://learn.microsoft.com/en-us/azure/openshift/troubleshoot for more details.")
```

Comment:
Can you update the link to this PR when it lands - https://github.com/MicrosoftDocs/azure-docs/pull/117572

Reply:
Updated